services/bundle_analysis/report.py

```diff
@@ -234,7 +234,7 @@ def process_upload(
         """
         commit_report: CommitReport = upload.report
         repo_hash = ArchiveService.get_archive_hash(commit_report.commit.repository)
-        storage_service = get_storage_client()
+        storage_service = shared.storage.get_appropriate_storage_service(commit.repoid)
         bundle_loader = BundleAnalysisReportLoader(storage_service, repo_hash)
```

Comment:
I’m wondering why this is not using `ArchiveService` directly? As that is pretty much just `storage_service + repo_hash`